.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+install:
+  - "pip install -r requirements.txt"
+  - "pip install -r test-requirements.txt"
+  - "pip install flake8"
```

Comment:
This line is not needed anymore, because we are installing flake8 in test-requirements.txt.